### DISCUSSION TITLE
Minimum needed values for ConformalIntervals?

### DISCUSSION THREAD

**Discussion by linaamic** [alan-turing-institute/sktime#3284] [Q&A]
Hi all, 

I have encountered some issues using ConformalIntervals.
For Exponential Smoothing, I have been able to get some intervals. However, I get an error when calling predict_interval when my training set (apparently) is too small. Fitting the forecaster beforehand and predicting point forecasts works fine! I haven't been able to identify the exact amount of data I need. 
Is there a set minimum amount of data points needed for using ConformalIntervals? 

Also wondering if this could be linked to #2889 

Any help is much appreciated

**Comment by fkiraly** [comment]
Yes, this is probably another instance pf https://github.com/alan-turing-institute/sktime/issues/2889, see potential workaround there.

The problem arises when you have nested sub-setting, i.e., if the wrapped estimator splits the data, so you end up splitting and then splitting again and then splitting again. The suspicion is that this problem arises if you end up splitting too much, having only 1 or 0 data points in the smallest split arising.

If the theory is correct, unfortunately, there is no "universal minimum", since the minimum might depend on how and how often the wrapped estimators split, internally.

The settings in #2889, e.g,. `initial_window=12, sample_frac=0.1`, might work for you?

If not, self-contained code (incl imports) would be appreciated, and a bug report!
